src/main/java/com/mongodb/Read.java

```diff
@@ -0,0 +1,70 @@
+package com.mongodb;
```

Comment:
No license header.

Also package name convention: `com.mongodb` is used for the driver libraries - normally I'd suggest an alternative for example code. For example we use `tour` for our Quick Start code, but it may be too late for that change.

Reply:
You mean `com.mongodb.tour`?
Would `com.mongodb.quickstart` be OK as well?

Reply:
@jdrumgoole Do we *really* need a license header? I find this overkill for what we are doing. What's your opinion?

Reply:
Package has been updated to `com.mongodb.quickstart` to match the series name.